### ISSUE TITLE
NRI pod can't access the /dev/isst_interface

### ISSUE THREAD

**Issue by changzhi1990** [containers/nri-plugins#101]
Hi, all. 

Based on my testing about the sst feature in the topology-aware policy. I found that there some problems in the NRI pod.

**The NRI pod can't find the `/host/dev/isst_interface` device.** 

![image](https://github.com/containers/nri-plugins/assets/12726668/ddbbf37b-3a57-4ce9-9d37-7eba14021b18)

After some research, I add these lines to the NRI daemonset. 
![image](https://github.com/containers/nri-plugins/assets/12726668/d4141fc2-c744-4726-9f4f-c5750e27b861)

Then, the NRI pod has no permission to access this device:
```
W0804 01:56:49.287017       1 system.go:297] failed to get SST info for package 0: failed to read SST PP info: Mbox command failed with failed to open isst device "/host/dev/isst_interface": open /host/dev/isst_interface: operation not permitted
```

After that, I noticed that there are some securitycontext in the daemonset file and I modified it:
![image](https://github.com/containers/nri-plugins/assets/12726668/a951ee4c-fa19-4570-becf-a3d56bc6aac3)

I added the `privileged: true` into it and I commented the next two lines. At last, the NRI can access the sst device:
![image](https://github.com/containers/nri-plugins/assets/12726668/d4dc428f-6ef6-4436-a446-59861a048653)

So does my approach was correct?

**Comment by marquiz** [comment]
Yes, looks about right. Running privileged should be an option (default to `false`) in the Helm chart

**Comment by changzhi1990** [comment]
> Yes, looks about right. Running privileged should be an option (default to `false`) in the Helm chart

Do we need to create a pr to fix it?

**Comment by changzhi1990** [comment]
> Yes, looks about right. Running privileged should be an option (default to `false`) in the Helm chart

Do we need to create a pr to fix it?

**Comment by marquiz** [comment]
> Do we need to create a pr to fix it?

Yes, we need that. Add new options to the Helm chart to enable privileged mode and mount the host-dev
